chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -280,25 +267,19 @@ impl PartialWitnessActor {
     ) -> Result<(), Error> {
         tracing::debug!(target: "client", ?partial_witness, "Receive PartialEncodedStateWitnessMessage");
 
-        let signer = match self.my_signer.get() {
-            Some(signer) => signer,
-            None => {
-                return Err(Error::NotAValidator(format!("handle partial encoded state witness")));
-            }
-        };
+        let signer = self
+            .my_signer
+            .get()
+            .ok_or(Error::NotAValidator(format!("handle partial encoded state witness")))?;
 
-        // Validate the partial encoded state witness.
+        // Validate the partial encoded state witness and forward the part to all the chunk validators.
         if validate_partial_encoded_state_witness(
             self.epoch_manager.as_ref(),
             &partial_witness,
             &signer,
             &self.store,
         )? {
-            // Store the partial encoded state witness for self.
```

Comment:
I guess this is no longer needed because we anticipate receiving via self-message instead?

Reply:
Yep!